src/Command/Ide/IdeCommandBase.php

```diff
@@ -12,10 +13,7 @@
  */
 abstract class IdeCommandBase extends CommandBase {
 
-  /**
-   * @var string
-   */
-  private $phpVersionFilePath;
+  use IdeCommandTrait;
```

Comment:
Need to move these methods to a trait in order to reuse them in the Pull command. Generally, I'm on a mission to move a lot of the code from command "base" classes into traits so they can be reused.